src/Makefile

```diff
@@ -12,14 +12,16 @@ endif
 CPPO_ARGS := $(shell ocaml configure.ml)
 CPPO = cppo $(CPPO_ARGS)
 
-OCAMLC = ocamlfind ocamlc -pp "$(CPPO)" -g -package bytes
-OCAMLOPT = ocamlfind ocamlopt -pp "$(CPPO)" -g -package bytes
+OCAMLC = ocamlfind ocamlc -pp "$(CPPO)" -g -package bytes -bin-annot
+OCAMLOPT = ocamlfind ocamlopt -pp "$(CPPO)" -g -package bytes -bin-annot
```

Comment:
extlib has to be backwards compatible as much as possible, need to enable this conditionally based on ocaml version